src/index.html

```diff
@@ -1,13 +1,368 @@
+                  class="bicycle__photo"
+                  >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="bicycle__description">
+              <h2 class="bicycle__name">Hydraulic disc brakes & Lightweight</h2>
+              <p class="bicycle__name-description">Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.</p>
+
+            </div>
+          </div>
+          <div class="grid grid--desktop">
+            <div class="grid__item
+            grid__item--2-5 grid__item--5-8">
+              <a href="#" class="button">Explore</a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contacts" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-5">
+              <form action="#">
+                <input
+                  name="name"
+                  type="text"
+                  placeholder="Name"
+                  class="contacts__field form-field"
+                >
+                <input
+                  name="email"
+                  type="email"
+                  placeholder="Email"
+                  class="contacts__field form-field"
+                >
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="
+                    contacts__field--last
+                    contacts__field
+                    form-field
+                    form-field--textarea"
+                >
+                </textarea>
+
+                <button class="button button--primary" id="button">
+                  Send
+                </button>
+
+              </form>
+            </div>
+            <div class="grid__item grid__item--8-9">
+              <div class="contacts__info">
+                <div class="contacts__info-title">
+                  Phone
+                </div>
+                <a  class="contacts__info-link" href="tel:+1 234 5555-55-55">
+                  +1 234 5555-55-55
+                </a>
+              </div>
+              <div class="contacts__info">
+                <div class="contacts__info-title">
+                  Email
+                </div>
+                <a  class="contacts__info-link" href="mailto:hello@miami.com">
+                    hello@miami.com
+                </a>
+              </div>
+
+              <div class="contacts__info">
+                <div class="contacts__info-title">
+                  Address
+                </div>
+                <a  class="contacts__info-link" href="https://goo.gl/maps/qT2RgkogJ9yZMHXi9">
```

Comment:
Checklist | point 9 - we should load this link in new page (add atribute target)